src/library/Uno.Material/Styles/Controls/NavigationView.xaml

```diff
@@ -0,0 +1,1273 @@
+														VerticalContentAlignment="Stretch"
+														HorizontalContentAlignment="Stretch"
+														Grid.Row="5" />
+
+										<!-- Left nav list -->
+										<NavigationViewList x:Name="MenuItemsHost"
+															Grid.Row="7"
+															Margin="0,0,0,20"
+															SelectionMode="Single"
+															IsItemClickEnabled="True"
+															HorizontalAlignment="Stretch"
+															SelectedItem="{TemplateBinding SelectedItem}"
+															ItemTemplate="{TemplateBinding MenuItemTemplate}"
+															ItemTemplateSelector="{TemplateBinding MenuItemTemplateSelector}"
+															ItemContainerStyle="{TemplateBinding MenuItemContainerStyle}"
+															ItemContainerStyleSelector="{TemplateBinding MenuItemContainerStyleSelector}"
+															SingleSelectionFollowsFocus="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.SingleSelectionFollowsFocus}" />
+
+										<ContentControl x:Name="FooterContentBorder"
+														IsTabStop="False"
+														VerticalContentAlignment="Stretch"
+														HorizontalContentAlignment="Stretch"
+														Grid.Row="8" />
+
+										<NavigationViewItem x:Name="SettingsNavPaneItem"
+															Grid.Row="9"
+															Icon="Setting" />
+									</Grid>
+								</SplitView.Pane>
+
+								<SplitView.Content>
+									<Grid x:Name="ContentGrid">
+										<Grid.RowDefinitions>
+											<RowDefinition Height="Auto" />
+											<RowDefinition Height="*" />
+										</Grid.RowDefinitions>
+
+										<ContentControl x:Name="HeaderContent"
+														MinHeight="{StaticResource PaneToggleButtonHeight}"
+														IsTabStop="False"
+														Margin="4,4,0,0"
+														Content="{TemplateBinding Header}"
+														ContentTemplate="{TemplateBinding HeaderTemplate}"
+														VerticalContentAlignment="Stretch"
+														HorizontalContentAlignment="Stretch"
+														Style="{StaticResource NavigationViewTitleHeaderContentControlTextStyle}" />
+
+										<ContentPresenter AutomationProperties.LandmarkType="Main"
+														  Grid.Row="1"
+														  win:Margin="-8,0,0,0"
+														  wasm:Margin="-8,0,0,0"
+														  Padding="0"
+														  Content="{TemplateBinding Content}" />
+									</Grid>
+								</SplitView.Content>
+							</SplitView>
+
+						</Grid>
+
+						<!--
+							Notes:
+							- InternalVisibleBoundsPadding is added to this control to make this template compatible
+							  with notched devices by default. This behavior is not present in Microsoft's default UWP default.
+
+							- Canvas.ZIndex is not supported in Uno Grid yet, see https://github.com/unoplatform/uno/issues/325
+						-->
+						<Grid x:Name="PaneToggleButtonGrid"
+							  Margin="0,0,0,8"
+							  HorizontalAlignment="Left"
+							  VerticalAlignment="Top"
+							  Canvas.ZIndex="100"
+							  xmlns:uBehaviors="using:Uno.UI.Behaviors">
```

Comment:
i think ubehaviors isn't used. (same for the one on top of the page)

Reply:
yup, uno pasta